.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
++**I'm submitting a ..**
++- [ ] Bug Report
++- [ ] Feature Request
++
++**Current behavior:**
++<!-- Describe the bug and how it creeped in. -->
```

Comment:
If you asked me "how it creeped in," I would try to look for the specific commit that changed the line the triggered the bug, and I don't think that's what we expect here. I'd just leave it at "Describe the bug."